base/array.jl

```diff
@@ -1596,14 +1596,14 @@ cat(n::Integer, x::Integer...) = reshape([x...], (ntuple(x->1, n-1)..., length(x
 """
     findnext(A, i::Integer)
 
-Find the next linear index >= `i` of a non-zero element of `A`, or `0` if not found.
+Find the next linear index >= `i` of a true element of `A`, or `0` if not found.
 
 # Examples
```

Comment:
It would make sense to add an example using `equalto`, since people will likely look at this docstring when they need this feature (same for other docstrings). Or just mention it in the description.